src/main/java/com/gotocompany/firehose/evaluator/DefaultGrpcPayloadEvaluator.java

```diff
@@ -0,0 +1,10 @@
+package com.gotocompany.firehose.evaluator;
+
+import com.google.protobuf.Message;
+
+public class DefaultGrpcPayloadEvaluator implements PayloadEvaluator<Message> {
+    @Override
+    public boolean evaluate(Message payload) {
+        return true;
```

Comment:
So now the default behavior is to retry on failures. So what if the customer wants to configure that for any error he don't want it to get retried. 
Option 1. We could do this by having another implementation and provide that implementation. 
```
SINK_GRPC_RESPONSE_RETRY_STRATEGY = ALWAYS/NEVER/CEL
```
Option 2. To reduce the number of config variables, we can define a convention that they can provide cel expression which is not going to be true for all the cases. e.code > 99999

Reply:
As discussed, lets use the CEL expression `false` if we want to disable this. Lets update this in the doc explicitly so that if someone wants to disable retry, they will be aware of this.

Reply:
I have updated the docs and mentioned the retry all and no retry case in the docs. Also I removed the implementation of `DefaultGrpcPayloadEvaluator`. We will directly use the CEL expression config as false and true to enable the cases mentioned previously